### PR TITLE
fix: Clear console errors

### PR DESCRIPTION
**Pull request by acezard**
Clearing small bugs that polluted the console. There is still one:

image

But it's not possible to fix it from cozy-notes, it's a cozy-sharing issue.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -57,7 +57,7 @@
   "dependencies": {
     "@atlaskit/icon": "21.1.0",
     "@atlaskit/media-core": "32.2.0",
-    "@material-ui/core": "4.12.3",
+    "@material-ui/core": "4.11.4",
```
Comment (Crash--) [inline package.json:60]: why this downgrade?

File: src/components/notes/EditorCorner.jsx [hunk 2]
```diff
@@ -7,13 +7,22 @@ import { useI18n } from 'cozy-ui/transpiled/react/I18n'
...
+})
+
 const EditorCorner = ({ doc, isPublic, isReadOnly, title }) => {
   const { t } = useI18n()
   if (!isPublic) return <SharingWidget file={doc.file} title={title} />
   else if (isReadOnly) {
     return (
       <Tooltip title={t('Notes.Editor.read_only')}>
-        <Icon icon="lock" color="var(--primaryTextColor)" />
+        <ForwardedIcon icon="lock" color="var(--primaryTextColor)" />
```
Comment (Crash--) [inline src/components/notes/EditorCorner.jsx:25]: Why not updating cozy-ui in order to forwardRef by default on Icon? 🤔

**Comment by Crash--** [comment]
Did you created the issue of cozy-sharing side? It is already created so you can link to it form here?

File: package.json (hunk 1 above)
Comment (acezard) [reply package.json:60]: it is required with the version of cozy-ui we use

File: src/components/notes/EditorCorner.jsx (hunk 2 above)
Comment (acezard) [reply src/components/notes/EditorCorner.jsx:25]: I don't remember why I didn't choose to update cozy-ui, maybe there were more problems. I can do that though

**Comment by acezard** [comment]
màj cozy-ui
issue sur cozy-ui si forwardref

**Comment by acezard** [comment]
Update: cozy-ui update is too complex, leaving it as is for now

File: src/components/notes/EditorCorner.jsx (hunk 2 above)
Comment (Crash--) [reply src/components/notes/EditorCorner.jsx:25]: Did you created the issue on cozy-ui? Can you link it in the code?

Comment (acezard) [reply src/components/notes/EditorCorner.jsx:25]: I didn't create an issue yet, the build is failing with the latest cozy-ui for a different reason (konnector related)

Comment (acezard) [reply src/components/notes/EditorCorner.jsx:25]: I mean, I'm not sure the forwardRef error exists with the latest cozy-ui (that doesn't work on cozy-notes right now, I prefered delivering the fixes instead)

Comment (acezard) [reply src/components/notes/EditorCorner.jsx:25]: I will check on the latest version after cozy-notes is delivered

Comment (acezard) [reply src/components/notes/EditorCorner.jsx:25]: https://github.com/cozy/cozy-ui/issues/1998

**Review by Crash-- — APPROVED** [review]

**Pull request merged**